### PR TITLE
[15.0] [MIG] fleet_vehicle_inspection: Migration to 15.0

### PR DESCRIPTION
**Pull request by rodrigonevest**
cc @marcelsavegnago @kaynnan @augustodinizl @douglascstd

### CODE REVIEW

File: fleet_vehicle_inspection/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Copyright 2020 - TODAY, Escodoo
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Fleet Vehicle Inspection",
+    "summary": """
+        This module extends the Fleet module allowing the registration
+        of vehicle entry and exit inspections.""",
+    "version": "15.0.3.0.1",
```
Comment (kaynnan) [inline fleet_vehicle_inspection/__manifest__.py]: "version": "15.0.3.0.1" -> "version": "15.0.1.0.0"

**Review by kaynnan — COMMENTED** [review]
LGTM

File: fleet_vehicle_inspection/__manifest__.py [hunk 2]
```diff
@@ -6,17 +6,16 @@
...
-    "version": "14.0.3.0.1",
+    "version": "15.0.3.0.1",
     "license": "AGPL-3",
     "category": "Human Resources",
     "author": "Escodoo,Odoo Community Association (OCA)",
     "maintainers": ["marcelsavegnago"],
     "images": ["static/description/banner.png"],
     "website": "https://github.com/OCA/fleet",
-    "depends": ["fleet"],
+    "depends": ["fleet", "web"],
```
Comment (ivantodorovich) [inline fleet_vehicle_inspection/__manifest__.py]: The `web` explicit dependency is not needed. Any reason to add it?

File: fleet_vehicle_inspection/__manifest__.py [hunk 3]
```diff
@@ -6,17 +6,16 @@
     "summary": """
         This module extends the Fleet module allowing the registration
         of vehicle entry and exit inspections.""",
-    "version": "14.0.3.0.1",
+    "version": "15.0.3.0.1",
```
Comment (ivantodorovich) [inline fleet_vehicle_inspection/__manifest__.py]: This should be `15.0.1.0.0`

File: fleet_vehicle_inspection/models/fleet_vehicle_inspection_line.py [hunk 4]
```diff
@@ -56,7 +54,7 @@ class FleetVehicleInspectionLine(models.Model):
     )
 
     inspection_line_image_ids = fields.One2many(
-        "fleet.vehicle.inspection.line.image", "inspection_line_id", string="Images"
+        "fleet.vehicle.inspection.line.image", "inspection_line_id"
```
Comment (ivantodorovich) [inline fleet_vehicle_inspection/models/fleet_vehicle_inspection_line.py]: Not sure removing this field's `string` makes sense.
The auto generated string is too long.

File: fleet_vehicle_inspection/static/src/scss/backend.scss [hunk 5]
```diff
@@ -1,62 +0,0 @@
...
-                width: auto;
-            }
-
-            .o_form_image_controls {
-                @include o-hover-opacity(0.7, 1);
-                padding: 2%;
-            }
-        }
-    }
-}
```
Comment (ivantodorovich) [inline fleet_vehicle_inspection/static/src/scss/backend.scss]: Has the removal of these styles been checked?

I didn't like having these, to be honest, so I'm happy if we can remove it.. but are you sure there's no bad side effects?

Could you compare the screenshots of the view in v14 and now?

File: fleet_vehicle_inspection/__manifest__.py [hunk 6]
```diff
@@ -13,7 +13,7 @@
     "maintainers": ["marcelsavegnago"],
     "images": ["static/description/banner.png"],
     "website": "https://github.com/OCA/fleet",
-    "depends": ["fleet", "web"],
+    "depends": ["fleet"],
```
Comment (ivantodorovich) [inline fleet_vehicle_inspection/__manifest__.py:16]: This should be squashed with the commit that added the `web` dependency

File: fleet_vehicle_inspection/static/src/scss/backend.scss (hunk 5 above)
Comment (rodrigonevest) [reply fleet_vehicle_inspection/static/src/scss/backend.scss]: Yes, it has been checked and there are no side effects. Here are the prints of the two screens:

Version 15 - local
image

Version 14 - runboat
image

**Review by ivantodorovich — APPROVED** [review]

**Review by marcelsavegnago — APPROVED** [review]

**Pull request merged**